lib/cli.js

```diff
@@ -24,20 +14,17 @@ var cmd = args[0];
 
 function pre() {
   if (opts.version) {
-    console.log(pkg.version);
-    return;
+    return console.log(pkg.version);
```

Comment:
then all the returns should be on there own line like https://github.com/yeoman/yo/blob/master/lib/cli.js#L59 and https://github.com/yeoman/yo/blob/master/lib/cli.js#L66